### PR TITLE
created wirebond class

### PR DESCRIPTION
**Pull request by joelgrebel**

### CODE REVIEW

**Comment by edumur** [comment]
Good work !
Many comments:
- When you multiply the bonding wire matrix in the klopfenstein_discretization file (we should changed its name by the way) like in the line 309, it should be [[1., 1j_o_self.L_b], [0., 1.]].
- I know it was my idea to create a new class but now I see the result I am not sure anymore... There is a lot of code duplication and I really don't like it. What about keeping one Klopfenstein class and add an optional parameter L_b which is None by default ?

File: klopfenstein_discretization.py [hunk 1]
```diff
@@ -78,9 +78,11 @@ def ljpa_external_discretization(self, f, z, prod, zl, as_theory):
...
         M = self.matrix_chain(o*prod, z)
 
-        # We end the chain by two elements:
-        # 1 - a load impedance to the ground
-        # 2 - a huge impedance to the circuit
+        # We end the chain by three elements:
+        # 1 - a wirebond inductance
+        # 2 - a load impedance to the ground
+        # 3 - a huge impedance to the circuit
+        M = np.dot(M, np.array([[1, 1j*o*self.L_b],[0, 1]]))
```
Comment (edumur) [inline klopfenstein_discretization.py]: Should 1. and 0.

**Comment by edumur** [comment]
Better ;) .
I think reflection_discretization and external_discretization should also be modified.
Furthermore the matrix multiplication must be done only when L_b is not None so add a condition.

**Comment by joelgrebel** [comment]
The results are identical for L_b=0. and L_b=None condition. Its just the multiplication of an identity matrix for L=0. Do you think it will significantly speed up the computation? Otherwise I think the code is cleaner this way.

**Comment by edumur** [comment]
Yep you right for L_b=0. (pretty sure None type cannot do math).
I did speed test and surprisingly, we have the same speed with and without adding a condition.
I am suspecting numpy or fortran to not perform the matrix multiplication if one of the matrix is equal to identity.
Anyway good job ;) .

**Pull request merged**